src/routes/interested/index.js

```diff
@@ -60,14 +61,10 @@ export default class Interested extends Component {
           <PlaceNumber reserved number={placeNumber} />
         </div>
         <div>
-          <p className={style.blueHint}>
-            <span className={style.blueHintLine}>
-              Jednak nie przyjedziesz?
-            </span>
-            <span className={style.blueHintLine}>
-              Daj skorzystać komuś innemu.
-            </span>
-          </p>
+          <BlueHint
```

Comment:
I would rather have 2 Hint components here than passing lines:
```
  <Hint blue>
    Jednak nie przyjedziesz?
  </Hint>
  <Hint blue>
    Daj skorzystać komuś innemu.
  </Hint>
```
This way it's super extensible and allows further enhancing parts of text.